resources/letters/models/letter_base.yml

```diff
@@ -41,7 +41,20 @@ allOf:
       return_envelope:
         oneOf:
           - type: boolean
-          - type: string
+          - type: object
```

Comment:
Could we make this inline object a `$ref` to `return_envelope.yml`? It's not a huge impact here, but it'll be nice when we eventually turn this into an SDK generation version.